### DISCUSSION TITLE
Clarify error message styling

### DISCUSSION THREAD

**Discussion by CobaltCause** [rust-lang-nursery/api-guidelines#250] [API Guidelines]
In [this section](https://rust-lang.github.io/api-guidelines/interoperability.html#examples-of-error-messages), it's hinted that error messages should not assume they are the beginning of the sentence/message/log line/whatever and always start with a lowercase letter (unless the first word happens to be an initialism/acronym). I think it would be a good idea to codify this in a paragraph explaining why they're written this way which would explicitly make this a requirement to be considered to be following the guidelines.

**Comment by Enet4** [comment]
Might be worth coordinating the discussion with the knowledge from the error handling working group (see for example rust-lang/project-error-handling#27).

**Reply by CobaltCause** [reply]
Oh cool, didn't know this existed :sweat_smile:

**Comment by matklad** [comment]
Note that this is currently codified in stdlib docs: https://doc.rust-lang.org/stable/std/error/trait.Error.html

>Error messages are typically concise lowercase sentences without trailing punctuation:

**Reply by CobaltCause** [reply]
lol I must have missed that, thanks for pointing it out
